pontus_autonomy/pontus_autonomy/prequalification_run.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import rclpy
+import numpy as np
+
+from pontus_autonomy.base_run import BaseRun
+
+# Tasks
+from pontus_autonomy.tasks.gate_task import GateTask
+from pontus_autonomy.tasks.vertical_marker_task import VerticalMarkerTask
+from pontus_autonomy.tasks.surface_task import SurfaceTask
+
+class PrequalificationRun(BaseRun):
+
+    def __init__(self):
+        super().__init__("prequalification_run")
+
+        print("Starting Prequalification Run")
+
+        # Gate Task
+        result = self.run_task(GateTask)
```

Comment:
Outside of prequal, how should information, such as the direction to circumnavigate the buoy, which is discovered during `self.run_task(GateTask)`, be passed back to the Run?